### DISCUSSION TITLE
Is Node.js aware of, affected by, mitigating  CVE-2022-2295?

### DISCUSSION THREAD

**Discussion by guest271314** [nodejs/help#4957] [Q&A] [answered]
Is Node.js, dependent on v8, aware of, affected by, mitigating [CVE-2022-2295](https://crbug.com/1336869)?

**Comment by bnoordhuis** [comment, marked as answer]
It requires running untrusted code so it doesn't fall under node's threat model (which is different from Chromium's threat model, which of course is very much about running untrusted code.)
